### PR TITLE
add reference to GNU/Linux, add examples for less command

### PR DESCRIPTION
**Pull request by codophobia**
https://github.com/linkedin/school-of-sre/issues/37
https://github.com/linkedin/school-of-sre/issues/76

This pull request is in reference to the above issues that were opened.

### CODE REVIEW

File: courses/linux_basics/command_line_basics.md [hunk 1]
```diff
@@ -326,11 +326,14 @@ More command displays the contents of a file or a command output,
 displaying one screen at a time in case the file is large (Eg: log files).
 It also allows forward navigation and limited backward navigation in the file.
 
+![](images/linux/commands/image10.png)
```
Comment (ritwik12) [inline courses/linux_basics/command_line_basics.md:329]:
```suggestion
![](images/linux/commands/image33.png)
```

Comment (ritwik12) [reply courses/linux_basics/command_line_basics.md:329]: @codophobia I guess you mistyped the image number here.

**Review by ritwik12 — CHANGES_REQUESTED** [review]
@codophobia I am not sure why we are adding the same image twice.

`courses/linux_basics/images/image33.png` is not needed

File: courses/linux_basics/command_line_basics.md (hunk 1 above)
Comment (codophobia) [reply courses/linux_basics/command_line_basics.md]: @ritwik12 That image was added twice by mistake. I have corrected the image number also. 👍

**Pull request merged**